src/components/Input.jsx

```diff
@@ -0,0 +1,16 @@
+const Input = ({ object, setObject, field, text, placeholder }) => {
+  return (
+    <div className="w-full">
+      <div className="font-medium text-sm mb-1">{text}</div>
+      <input
+        className="border border-parts-gray-400 rounded w-full bg-transparent px-2 py-1"
+        type="text"
+        placeholder={placeholder}
+        value={object[field]}
+        onChange={(e) => setObject({ ...object, [field]: e.target.value })}
```

Comment:
on change should be handled in parent and just passed down, this caused a bunch of problems later on with hackathon website :sob: